packages/apps/plugins/plugin-markdown/src/MarkdownPlugin.tsx

```diff
@@ -242,17 +240,19 @@ export const MarkdownPlugin = (): PluginDefinition<MarkdownPluginProvides> => {
                   </MainLayout>
                 );
               } else if (
-                'model' in data &&
-                isEditorModel(data.model) &&
+                // TODO(burdon): Replace model with object ID.
+                // 'model' in data &&
+                // isEditorModel(data.model) &&
                 'properties' in data &&
                 isMarkdownProperties(data.properties)
               ) {
-                const main = <EditorMain model={data.model} extensions={extensions} />;
-                if ('view' in data && data.view === 'embedded') {
-                  return <EmbeddedLayout>{main}</EmbeddedLayout>;
-                } else {
-                  return <MainLayout>{main}</MainLayout>;
-                }
+                return null;
+                // const main = <EditorMain extensions={extensions} />;
```

Comment:
this is why we need stories for the editor w/o automerge, so that we can ensure that works in non-automerge scenarios in composer (e.g. local files)

Reply:
Added TODO. i do see this as important once we've landed the AM branches.